backend/fms_core/models/property_value.py

```diff
@@ -0,0 +1,59 @@
+import reversion
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+from .tracked_model import TrackedModel
+from .property_type import PropertyType
+
+from ._utils import add_error as _add_error
+
+from ..schema_validators import PROPERTY_VALUE_VALIDATOR
+
+import json
+
+
+__all__ = ["PropertyValue"]
+
+@reversion.register()
+class PropertyValue(TrackedModel):
+    value = models.JSONField("Property value", validators=[PROPERTY_VALUE_VALIDATOR],
+                                      help_text="Property value")
+    property_type = models.ForeignKey(PropertyType, on_delete=models.PROTECT, related_name="property_type",
+                                  help_text="Property type")
+
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
```

Comment:
on_delete=models.PROTECT instead of CASCADE maybe ? i don't see much case where this would be important but between both i would prefer not to lose all property values related to a content_type ...

Reply:
I prefer CASCADE over PROTECT. If we are to delete a model (which is done after lots of reflexion), we dont want to have any dangling objects related that are broken, nor do we want to go on to manually delete all of these objects in our migration.

Reply:
That could be true in some contexts but here we have meta data that we'll never want to delete and even less in bulk cascade like that. If we ever need to move that information we'll remove the PROTECT once the data is safe. I do not see any good reason to use cascade here. This could only lead to mistakenly deleting this information.